### ISSUE TITLE
Fail to load config in rector-prefixed

### ISSUE THREAD

**Issue by snapshotpl** [rectorphp/rector#3748]
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | e.g. v0.7.50              |
| Installed as   | prefixed Rector PHAR by composer                      |

```
 [ERROR] File path                                                              
         "phar:///var/www/vendor/rector/rector-prefixed/rector/packages/set/src/
         ../../../config/set/action-injection-to-constructor-injection.php" was 
         not found while creating "Symplify\SmartFileSystem\SmartFileInfo"      
         object.
```

Changing or clean config doesn't change anything.

**Comment by snapshotpl** [comment]
0.7.50 and 0.7.49 are affected. Downgrade to 0.7.48 works fine.

**Comment by TomasVotruba** [comment]
Note: Related to use of realpaths in sets resolving, see: https://github.com/rectorphp/rector/pull/3741

[`realpath()` does nto work in PHAR](https://stackoverflow.com/questions/27838025/how-to-get-a-phar-file-real-directory-within-the-phar-file-code)

**Comment by TomasVotruba** [comment]
Resolved in v0.7.53
